mod/legacy_urls/start.php

```diff
@@ -311,7 +311,7 @@ function legacy_urls_groups_forward($hook, $type, $result) {
 
 	if (is_numeric($page[0])) {
 		$group = get_entity($page[0]);
-		if (elgg_instanceof($group, 'group', '', 'ElggGroup')) {
+		if (elgg_instanceof($group, 'group', '')) {
```

Comment:
3rd param can also be removed as it isn't check in `elgg_instanceof` if it's empty